src/language.cpp

```diff
@@ -128,14 +127,25 @@ StringParameters::StringParameters()
 	this->Clear();
 }
 
+/**
+ * Ensure that this Parameters object can hold at least a certain number of parameters.
+ * @param num Minimum number of parameters.
+ */
+void StringParameters::ReserveCapacity(const int num)
+{
+	assert(num > 0);
```

Comment:
using `0` as minimum looks like a valid value to me, if you don't want that, state the condition of minimum more clearly in the documentation.

Reply:
If 0 was allowed we'd need to ad an `assert(num > 0)` in every function that calls this one because parameter indices are 1-based. Added a note to the documentation.